.circleci/config.yml

```diff
@@ -15,16 +15,25 @@ save-cache-osx: &save-cache-osx
       - /Users/distiller/project/node_modules
     key: osx-dependencies-{{ checksum "../package.json" }}
 android-secrets: &android-secrets
-  run: 
+  run:
     command: |
       git clone git@github.com:redbadger/pride-android-secrets.git
-      mkdir -p .gradle 
-      echo -e "keystore=$ANDROID_KEYSTORE_PATH\nkeystore.alias=$ANDROID_KEYSTORE_ALIAS\nkeystore.password=$ANDROID_KEYSTORE_PASSWORD\nkeystore.keypass=$ANDROID_KEYSTORE_PASSWORD\nversionCode=$CIRCLE_BUILD_NUM" >> .gradle/gradle.properties
-      echo -e "apiSecret=$FABRIC_BUILD_SECRET\napiKey=$FABRIC_API_KEY" >> ./app/fabric.properties  
+      mkdir -p .gradle
+      echo -e "keystore=$ANDROID_KEYSTORE_PATH\nkeystore.alias=$ANDROID_KEYSTORE_ALIAS\nkeystore.password=$ANDROID_KEYSTORE_PASSWORD\nkeystore.keypass=$ANDROID_KEYSTORE_PASSWORD\nversionCode=$CIRCLE_BUILD_NUM" > .gradle/gradle.properties
+android-release-secrets: &android-release-secrets
+  run:
+    command: |
+      git clone git@github.com:redbadger/pride-android-secrets.git
+      mkdir -p .gradle
+      echo -e "keystore=$ANDROID_RELEASE_KEYSTORE_PATH\nkeystore.alias=$ANDROID_RELEASE_KEYSTORE_ALIAS\nkeystore.password=$ANDROID_RELEASE_KEYSTORE_PASSWORD\nkeystore.keypass=$ANDROID_RELEASE_KEYSTORE_PASSWORD\nversionCode=$CIRCLE_BUILD_NUM" > .gradle/gradle.properties
+fabric-android-secrets: &fabric-android-secrets
+  run:
+    command: |
+      echo -e "apiSecret=$FABRIC_BUILD_SECRET\napiKey=$FABRIC_API_KEY" >> ./app/fabric.properties
 ruby-dependencies: &ruby-dependencies
   run:
     name: Download Ruby Dependencies
-    command: bundle check || bundle install --path vendor/bundle
```

Comment:
This check was causing issues after removing https://github.com/redbadger/pride-london-app/pull/157/files#diff-1d37e48f9ceff6d8030570cd36286a61L245 will revisit in different branch